scss/decanter.scss

```diff
@@ -1,20 +1,39 @@
-@charset "UTF-8";
+@charset 'UTF-8';
 
+///
+// Load Order
 //
-// DECANTER
+// 1. Pre-vendor settings
+// 2. Vendors
+// 3. Post-vendor settings
+// 4. Utilities (functions, mixins, extends)
+// 5. Core Elements
+// 6. Components
 //
 
-// Vendor
+// 1. Pre-vendor settings.
+@import 'variables/pre-vendor';
+
+// 2. Vendors.
+@import
+  "bourbon",
+  "neat",
+  "font-awesome",
+  "normalize";
+
+// 3. Post Vendor Settings.
 @import
-  'lib/bourbon',
-  'lib/neat',
-  'lib/normalize';
+  "variables/index";
```

Comment:
Variables had to be pulled out as it's own thing as they need to be accessible without bringing along any markup.